script/eslint-plugin-va/rules/prefer-web-component-library.js

```diff
@@ -0,0 +1,60 @@
+const MESSAGE =
+  '<{{ reactComponent }}> can be replaced by <{{ webComponent }}>';
+
+const getPropNode = (node, propName) =>
+  node.openingElement.attributes.find(n => n.name.name === propName);
+
+const telephoneTransformer = (context, node) => {
+  const componentName = node.openingElement.name;
+  const patternNode = getPropNode(node, 'pattern');
+  const notClickableNode = getPropNode(node, 'notClickable');
+  const international =
+    patternNode?.value.expression.property.name === 'OUTSIDE_US';
+
+  context.report({
+    node,
+    message: MESSAGE,
+    data: {
+      reactComponent: componentName.name,
+      webComponent: 'va-telephone',
+    },
```

Comment:
This is just to [fill in the message placeholders](https://eslint.org/docs/developer-guide/working-with-rules#using-message-placeholders)